src/tilemap/farmable.rs

```diff
@@ -0,0 +1 @@
+pub struct Farmable {}
```

Comment:
The idiomatic syntax for making a ZST like this is
`pub struct Farmable;`

Furthermore, I'm not sure whether or not this should really be in the `tilemap` module; Isn't farm behavior more generic?

Reply:
Or, not necessarily more generic but certainly better suited to a farm module.